jetbrains-ultimate/tst-201/software/aws/toolkits/jetbrains/services/rds/actions/CreateConfigurationActionTest.kt

```diff
@@ -13,23 +13,32 @@ import org.junit.Test
 import software.amazon.awssdk.services.rds.model.DBInstance
 import software.amazon.awssdk.services.rds.model.Endpoint
 import software.aws.toolkits.core.utils.RuleUtils
+import software.aws.toolkits.jetbrains.core.MockResourceCacheRule
 import software.aws.toolkits.jetbrains.core.credentials.MockCredentialsManager
 import software.aws.toolkits.jetbrains.core.region.MockRegionProvider
 import software.aws.toolkits.jetbrains.datagrip.CREDENTIAL_ID_PROPERTY
 import software.aws.toolkits.jetbrains.datagrip.REGION_ID_PROPERTY
+import software.aws.toolkits.jetbrains.services.rds.RdsDatasourceConfiguration
 import software.aws.toolkits.jetbrains.services.rds.auth.IamAuth
 import software.aws.toolkits.jetbrains.services.rds.jdbcMysql
 import software.aws.toolkits.jetbrains.services.rds.jdbcPostgres
 import software.aws.toolkits.jetbrains.services.rds.mysqlEngineType
 import software.aws.toolkits.jetbrains.services.rds.postgresEngineType
+import software.aws.toolkits.jetbrains.services.sts.StsResources
 
 class CreateConfigurationActionTest {
     @Rule
     @JvmField
     val projectRule = ProjectRule()
+
+    @Rule
+    @JvmField
+    val resourceCache = MockResourceCacheRule(projectRule)
+
     val port = RuleUtils.randomNumber()
     val address = RuleUtils.randomName()
     val username = "${RuleUtils.randomName()}CAPITAL"
+    val masterUsername = "${RuleUtils.randomName()}C"
```

Comment:
Why is there a C on the end?

Reply:
Same reason as above, to make it mixed case. In this case it's kind of weird and not needed, removed